api/API.md

```diff
@@ -39,29 +39,34 @@ The root endpoint for all API requests for version 1.0 of the API is `/api/v1`
 
 ### RecrudescenceTest
 
-Given an input `.csv` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
+Given an input `.xlsx` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
 
-*Method(s):* `PUSH`
+*Method(s):* `POST`
 
 *URL Endpoint:* `/api/v1/recrudescences`
 
-#### PUSH
+#### POST
 
 **Input Parameters (URL):**
 
 -   (optional) `iterations` (integer): The number of iterations to run the algorithm for on the data; higher values are more accurate but take longer to complete. Defaults to 10000.
 
 **Input Parameters (Data):**
 
--   `file` (.csv file, FormData): The file containing the testing data to be processed.
+-   `file` (.xlsx file, FormData): The file containing the testing data to be processed.
+
+**Input Parameters (URL):**
+
+-   (optional) `locirepeat` (list of integers): List of integers representing loci repeats. Each loci repeat must be individually entered for the request. Please ensure the correct amount of values are entered.
 
 **Example Usage:**
 
 In cURL:
 
 ```bash
-curl -F "file=@example.csv" http://localhost:5000/api/v1/recrudescences?iterations=10
+curl -F "file=@example.xlsx" "http://localhost:5000/api/v1/recrudescences?iterations=10&locirepeat=2&locirepeat=2&locirepeat=3"
 ```
+If you are using curl and are entering multiple query parameters, ensure you use quotes around the request
```

Comment:
This should already be known by people who are using curl and can easily be looked up, so I don't think we need to include it; if you still want to, I would reformat it slightly to be in italics/etc. (to make it clear it's a side-note)

Reply:
True. I will delete it.